solutions/usaco-667.mdx

```diff
@@ -2,65 +2,150 @@
 id: usaco-667
 source: USACO Silver 2016 December
 title: Cities & States
-author: Benjamin Qi
+author: Benjamin Qi, Kevin Sheng
 ---
 
 [Official Editorial (Java)](http://www.usaco.org/current/data/sol_citystate_silver_dec16.html)
 
-<LanguageSection>
+As the editorial mentions, we can store the number of times a given four-letter
+string appears in a `map`.
 
-<JavaSection></JavaSection>
+**Time Complexity:** $\mathcal{O}(N)$
 
+<LanguageSection>
 <CPPSection>
 
-As the editorial mentions, we can store the number of times a given four-letter
-string appears in a `map`.
+<Info>
 
-**Time Complexity:** $\mathcal{O}(N\log N)$ with `map`, $\mathcal{O}(N)$ with
-`unordered_map`.
+As the below solution uses `map`, the complexity is actually
```

Comment:
```suggestion
The time complexity of the below solution is actually
```

Reply:
i feel like mentioning `map` is necessary

Reply:
uhh i said "since it uses `map`" the following line